src/main/markdown/doc/latest/tutorial/style.md

```diff
@@ -32,55 +32,43 @@ When you used webAppCreator to create StockWatcher, it created the application s
 
 1.  Open the GWT module, StockWatcher/src/com/google/gwt/sample/stockwatcher/StockWatcher.gwt.xml.  Notice that the Standard theme is being used by default.
 
-```
-<?xml version="1.0" encoding="UTF-8"?>
-<module rename-to='stockwatcher'>
-  <!-- Inherit the core Web Toolkit stuff.                        -->
-  <inherits name='com.google.gwt.user.User'/>
-
+    ```xml
```

Comment:
the best is to remove the triple backtick and indent the code by `4 * (n + 1)` spaces, n being the depth of the list